packages/test-utils/src/legacy-cli/main.ts

```diff
@@ -286,16 +281,15 @@ async function directUploadRecording(
 }
 
 async function doUploadRecording(
-  dir: string,
```

Comment:
Thanks for verifying that it was okay for us to remove this. I wanted to with the earlier PR but it was just too big already.